cmd/internal/flag.go

```diff
@@ -38,7 +39,7 @@ var (
 	eventsRateLimitBurst int
 	// engine
 	enablePolicyException  bool
-	exceptionNamespace     string
+	exceptionNamespaces    policy.StringSlice
```

Comment:
It is better to use the `[]string` directly.

Reply:
done